contribute.md

```diff
@@ -0,0 +1,48 @@
+# Contribute
+We take pull requests!  Please read the [PHILOSOPHY.md](PHILOSOPHY.md) and search the issues before submitting a PR.
+
+## Create a fork!
+Create a fork on your own Github project (or your personal space)
+
+[Github Documentation on Forking a repo](https://help.github.com/articles/fork-a-repo/)
+
+## Stay in Sync
+
+### Configure upstream
+Configure Lightbulb as your upstream so you can stay in sync
+
+```bash
+git remote add upstream https://github.com/ansible/lightbulb.git
+```
+
+[More info on github.com](https://help.github.com/articles/configuring-a-remote-for-a-fork/)
+
+### Syncing a fork
```

Comment:
really like the why, who and best practice bit of this. It can be seen as trivial but I think it adds value and provides an understanding on why certain changes were made.